extensions/common/nlu-devops-common/utilities/index.ts

```diff
@@ -87,3 +57,45 @@ export async function getNLUToolRunner(): Promise<tr.ToolRunner> {
     return tl.tool(dotnetPath)
         .arg("nlu");
 }
+
+export async function installDotNetTool(packageName: string): Promise<any> {
+    const dotnetPath = tl.which("dotnet", false);
```

Comment:
Maybe add a private function so we don't call `tl.which` twice for the `getNLUToolRunner` method

Reply:
Make the return type boolean.